runtime/src/main/java/org/corfudb/runtime/object/StreamId.java

```diff
@@ -0,0 +1,35 @@
+package org.corfudb.runtime.object;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+import org.corfudb.runtime.CorfuRuntime;
+
+import javax.annotation.Nullable;
+import java.util.UUID;
+
+@EqualsAndHashCode
+@Getter
+@ToString
+public class StreamId {
+    @Nullable
+    private String name;
```

Comment:
let's transform it to Optional, as you suggested earlier

Reply:
I'll just go ahead and ignore the warking then